### PR TITLE
Build/package region meta

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7988
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] ~~Standalone app was ran and tested locally~~
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/veeam-enterprise/package.json [hunk 1]
```diff
@@ -65,5 +65,8 @@
     "@ovh-ux/manager-webpack-config": "^3.3.0 || ^4.0.0",
     "glob": "^7.1.6",
     "webpack-merge": "^4.2.2"
-  }
+  },
+  "regions": [
+    "EU"
```
Comment (marie-j) [inline packages/manager/apps/veeam-enterprise/package.json:70]:
```suggestion
    "EU",
    "US"
```

File: packages/manager/tools/sao-ovh-manager-app/saofile.js [hunk 2]
```diff
@@ -24,6 +24,16 @@ module.exports = {
           return `OVHcloud ${camelcase(name, { pascalCase: true })} app`;
         },
       },
+      {
+        name: 'regions',
+        message:
+          'To which region this product will be available (comma-separated values)',
```
Comment (marie-j) [inline packages/manager/tools/sao-ovh-manager-app/saofile.js:30]:
```suggestion
          'In which region this product will be available (comma-separated values)',
```

Comment (marie-j) [reply packages/manager/tools/sao-ovh-manager-app/saofile.js:30]: Can we have a multi selector list maybe instead ?

Comment (JeremyDec) [reply packages/manager/tools/sao-ovh-manager-app/saofile.js:30]: Multiselect list doesn't seems to be available through `sao` (even tested version `2.0.0-beta.1`)

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/tools/sao-ovh-manager-app/saofile.js (hunk 2 above)
Comment (antleblanc) [reply packages/manager/tools/sao-ovh-manager-app/saofile.js]: Seems to be available.

See:
- https://github.com/saojs/sao/blob/v1.7.0/package.json#L31
- https://github.com/SBoudrias/Inquirer.js/tree/v6.0.0#checkbox---type-checkbox

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/tools/sao-ovh-manager-app/saofile.js [hunk 3]
```diff
@@ -24,6 +24,14 @@ module.exports = {
           return `OVHcloud ${camelcase(name, { pascalCase: true })} app`;
         },
       },
+      {
+        type: 'checkbox',
+        choices: ['CA', 'EU', 'US'],
+        name: 'regions',
+        message:
+          'In which region this product will be available (comma-separated values)',
```
Comment (marie-j) [inline packages/manager/tools/sao-ovh-manager-app/saofile.js:32]:
```suggestion
          'In which region this product will be available',
```

**Review by antleblanc — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Pull request merged**